pages/repositories.html

```diff
@@ -10,18 +10,18 @@
 	 <meta name="author" content="Junior Developer Group" />
 	 <meta
 	   name="description"
-	   content="Manage projects with personalized dashboard"
+	   content="Junior Developer Group | Dashboard page containing all current and previous projects"
```

Comment:
I've updated the description here to give us a bit more info about this page and what it does.. this will be updated in our SEO sprint soon but for now we can leave it like this :)